.github/workflows/Dockerfile

```diff
@@ -15,13 +15,16 @@ COPY --from=FOUNDRY /usr/local/bin/cast /usr/local/bin/cast
 
 COPY --from=Z3 /usr/bin/z3 /usr/bin/z3
 
+ARG LLVM_VERSION
+
 RUN    apt-get update                                   \
     && apt-get install --yes software-properties-common \
     && add-apt-repository ppa:ethereum/ethereum
 
 RUN    apt-get update         \
     && apt-get upgrade --yes  \
     && apt-get install --yes  \
+            clang-${LLVM_VERSION} \
             cmake             \
             curl              \
             debhelper         \
```

Comment:
All of the `llvm-*` should also get `${LLVM_VERSION}` number here. Below, it's `llvm-14-dev` and `llvm-14-tools`. So we should default `LLVM_VERSION=14` in the `with-docker` action, and make sure to use that same version for all LLVM related tools.

Reply:
Also let's try to keep the original formatting as much as possible, everything lined up nicely. Makes the files much more "scannable".